.buildkite/hooks/pre-command

```diff
@@ -3,23 +3,23 @@
 set -eu
 
 # Only execute in the verify pipeline
-[[ "$BUILDKITE_PIPELINE_NAME" =~ verify$ ]] || exit 0
+[[ "$BUILDKITE_PIPELINE_NAME" =~ verify$ ]] || [[ "$BUILDKITE_PIPELINE_NAME" =~ validate/release$ ]] || [[ "$BUILDKITE_PIPELINE_NAME" =~ validate/adhoc$ ]]  || [[ "$BUILDKITE_PIPELINE_NAME" =~ validate/adhoc-canary$ ]] || exit 0
```

Comment:
Should we do this as a proper regex? `(verify|validate/(release|adhoc|canary))$`

Reply:
on main i did this.. forgot to change it: 

`[[ "$BUILDKITE_PIPELINE_NAME" =~ verify$ ]] || [[ "$BUILDKITE_PIPELINE_NAME" =~ validate/.* ]] || exit 0`